libflux/flux-core/src/formatter/mod.rs

```diff
@@ -727,29 +713,41 @@ impl Formatter {
         if !n.body.is_empty() {
             self.indent()
         }
+        self.write_rune(sep);
+        // format the block statements
+        self.statement_list(&n.body);
 
-        let mut prev: i8 = -1;
-        for (i, stmt) in n.body.iter().enumerate() {
-            let cur = stmt.typ();
+        if !n.body.is_empty() {
             self.write_rune(sep);
+            self.unindent();
+            self.write_indent()
+        }
+        self.format_comments(&n.rbrace);
+        self.write_rune('}')
+    }
 
+    fn statement_list(&mut self, n: &[Statement]) {
+        // &Vec<Statement>
```

Comment:
Can you change the name to `format_statement_list` to be consistent with the other methods that format things?
And also can this comment line be deleted?